tasks/managed/rh-sign-image-cosign/rh-sign-image-cosign.yaml

```diff
@@ -61,6 +65,16 @@ spec:
               name: $(params.secretName)
               key: REKOR_URL
               optional: true
+        - name: PUBLIC_KEY
+          valueFrom:
+            secretKeyRef:
+              name: $(params.secretName)
+              key: PUBLIC_KEY
+        - name: REKOR_PUBLIC_KEY
```

Comment:
This is new from the last iteration and likely isn't in the cosign secret currently. Can you message konflux-releng on slack and try to get it added?

Reply:
sure

Reply:
it should be in the vault now

Reply:
I don't think it is in the namespace
`an error happened during storing pipelineRun log managed-release-team-tenant:managed-6k64r: container "step-sign-image" in pod "managed-6k64r-rh-sign-image-cosign-pod" is waiting to start: CreateContainerConfigError` did you just add it to vault or ping releng to get it added?

Reply:
Hey @johnbieren `REKOR_PUBLIC_KEY` was added to the secrets in Vault. Could you please confirm if they are synced successfully? I don't think I have access to the tenant to check.

Also, shouldn't these be defined as well in int srv namespace?

Reply:
Well, for one I don't have access either, that's why I asked @midnightercz to talk to konflux-releng. For two, this task is a managed task, not an internal one (it is in `tasks/managed`) this means that it runs on the "public" clusters, not the internal ones where the srv namespace is. I do not even have access to the vault instance where the secrets for the "public" clusters are, so I don't know who added the key or how, but the e2e logs say it is not on the clusters

Reply:
Jindrich asked us, konflux-releng, and we have updated the secrets in Vault. I'll discuss this further with my team to check if there is something missing. Thanks!

Reply:
Okay. Yeah I mean I can't see the secrets or it in vault, so I can't confirm anything beyond the output I pasted. Was it added to stage and prod or just stage?

Reply:
it was added to both stage & production secret paths.

Reply:
@midnightercz @zanssa through troubleshooting something else, I learned how this works in stage. Turns out I can edit this secret in stage (not prod) and I can confirm `REKOR_PUBLIC_KEY` is not there. The current keys are:
AWS_ACCESS_KEY_ID
AWS_DEFAULT_REGION
AWS_SECRET_ACCESS_KEY
PUBLIC_KEY
SIGN_KEY

If you want to send it to me on slack or something, I will add it

Reply:
Thanks @johnbieren! That's my understanding as well based on the discussion with @jinqi7 in the slack thread related to this request.

Were you able to confirm if the updated secrets in the cluster are correct? (I understand that they were updated and different from the ones in Vault) In that case, could we please update the values in Vault (the path is in the slack thread - it should be the one at konflux release)? 

We might also need to confirm if REKOR_URL is needed as well for e2e tests so the pipeline won't fail with missing secrets. WDYT?

Reply:
Hmm. one more key that I see in stage & prod which is username. @midnightercz could you please provide Johnny with the required secrets?

Reply:
I don't have access to the vault secret since it is in the releng engine (I don't know how Jing does). It looks like all the secrets now used in this task are present though (must be from Jing updating it a few hours ago) so I am resolving this